lib/created_message_queue_handler.ts

```diff
@@ -291,22 +341,40 @@ export async function handleMessage(
     );
   }
 
+  const maybeUnblockedEmailNotification = maybeIsUnblockedEmail.chain(
+    _ => maybeEmailNotification
+  );
+
   //
   //  Webhook notification
   //
 
+  // check if the user has blocked webhook notifications sent from this service
+  const isBlockedWebhook = blockedChannels.has(BlockedChannelEnum.WEBHOOK);
```

Comment:
think of boolean identifiers as questions: is webhook blocked? -> `isWebhookBlocked` (also for consistency with `isWebhookEnabled`)